interpreter/binary/decode.ml

```diff
@@ -146,6 +146,11 @@ let elem_type s =
   | -0x10 -> FuncRefType
   | _ -> error s (pos s - 1) "invalid element type"
 
+let extern_kind s =
```

Comment:
Rename to elem_kind, move to Tables part.

Reply:
Done.